passes/techmap/zinit.cc

```diff
@@ -139,11 +139,26 @@ struct ZinitPass : public Pass {
 
 				cell->setPort(ID(D), sig_d);
 				cell->setPort(ID(Q), initwire);
+
+				/*if (cell->type.in(ID($_DFFSR_NNN_), ID($_DFFSR_NNP_), ID($_DFFSR_NPN_), ID($_DFFSR_NPP_),
+							ID($_DFFSR_PNN_), ID($_DFFSR_PNP_), ID($_DFFSR_PPN_), ID($_DFFSR_PPP_)))
+				{
+					// TODO: I think I need a $_DFFRS_* cell where R has priority over S...
```

Comment:
You mean "where S has priority over R" (R over S is the standard pretty much everywhere, including our own cells)

Reply:
Oops right you are. Let me change that. I had assumed that the `SR` moniker meant that `S` > `R`